### DISCUSSION TITLE
求助大佬，如何将注释按照颜色分类，且图片注释也能加入到笔记中去！

### DISCUSSION THREAD

**Discussion by guguanghui10** [windingwind/zotero-better-notes#651] [Note Templates]
现有模板中按照颜色分类，文中的图片注释并没有加入到笔记中去，求大佬写一个文字注释按照颜色分类，然后同时能够将图片注释也能加到笔记中的模板！！感谢！！达到一个类似的效果！！
![Snipaste_2023-08-24_20-47-32](https://github.com/windingwind/zotero-better-notes/assets/108522047/ccce4d5c-c651-4ed2-88ca-639b3183030e)

**Comment by windingwind** [comment]
你得提供你的模板代码

**Reply by LynnXie00** [reply]
他在说这个，对，就是我很久以前用的模板
https://github.com/windingwind/zotero-better-notes/discussions/406

我也是修改了别人的，对，就是你写的……
https://github.com/windingwind/zotero-better-notes/discussions/227

稍微改了一下，可以在新版运行了。但是不能自动导入图片。图片是手动导入。其实也可以了。

**Comment by AnZhiJJ** [comment]
利用zotero figure生成的图片链接或者自己生成一个图片链接导入即可。
